doc/guide/https.xml

```diff
@@ -30,11 +30,16 @@
 
     <para>Cockpit will load a certificate from the <code>/etc/cockpit/ws-certs.d</code>
       directory. It will use the last file with a <code>.cert</code> extension in
-      alphabetical order. The <literal>.cert</literal> file should contain at least two
-      OpenSSL style PEM blocks. First one or more <literal>BEGIN CERTIFICATE</literal>
-      blocks for the server certificate and the intermediate certificate authorities
-      and a last one containing a <literal>BEGIN PRIVATE KEY</literal> or similar.
-      The key may not be encrypted. For example:</para>
+      alphabetical order. The <literal>.cert</literal> file should contain
+      one or more OpenSSL style <literal>BEGIN CERTIFICATE</literal>
+      blocks for the server certificate and the intermediate certificate authorities.</para>
+
+    <para>The private key can either be contained in the same <code>.cert</code> file
+    as an additional <literal>BEGIN PRIVATE KEY</literal> or similar block, or in
+    a separate file with the same name as the certificate, but with a <code>.key</code>
+    suffix instead of <code>.cert</code>. The key may not be encrypted. For example,
```

Comment:
Since you're making changes here anyway, let's say "must not be".

Reply:
That got fixed in the recent force push.